tests/test_scalyr.py

```diff
@@ -367,16 +382,34 @@ def test_remove_log_target(monkeypatch, env, exc):
     rmtree.assert_called_with(os.path.join(agent.dest_path, container_id))
 
 
+SERVER_ATTRIBUTES_IN = {
```

Comment:
Isn't `SERVER_ATTRIBUTES_IN == SERVER_ATTRIBUTES_OUT`, so we can use only one dict?

Reply:
True. And this would be a more accurate test for what the template is now doing